assets/ui/spawnScreen.ui

```diff
@@ -8,100 +8,100 @@
         "contents" : [
             {
                 "type" : "UIBox",
-                "layoutInfo": {
+                "layoutInfo" : {
                     "width": 500,
                     "use-content-height": true,
                     "position-horizontal-center": {},
                     "position-vertical-center": {}
                 },
-                "content": {
-                    "type" : "ColumnLayout",
-                    "columns": 2,
-                    "verticalSpacing" : 8,
-                    "contents" : [
+                "content" :
                     {
```

Comment:
Due to having { on the next line, you got confused about the indentation as well. Lines 19-22 should be indented one level less. However, the content after line 23 is correctly indented since the brace on line 23 starts a new object *inside* the list `contents` that is started on line 22.